docs/src/design/stack/crypto_ops.md

```diff
@@ -150,3 +150,29 @@ To keep the degree of the constraints low, a number of intermediate values are u
 
 The effect on the rest of the stack is:
 * **Left shift** starting from position $16$.
+
+## RCombBase
```

Comment:
nit: In these docs we usually capitalize operation names. So, we should use `RCOMBBASE` instead of `RCombBase`.

Reply:
Changed